<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home_DevWid</class>
 <widget class="QWidget" name="Home_DevWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>666</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>常用操作</string>
     </property>
     <widget class="QPushButton" name="serialBtn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>串口检测</string>
      </property>
     </widget>
     <widget class="QPushButton" name="outletBtn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>执行</string>
      </property>
     </widget>
     <widget class="QComboBox" name="outletBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>输出位全关</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>输出位全开</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="eleClearBtn">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>电能清除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="factoryBtn">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>80</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>恢复出厂设置</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>140</height>
      </size>
     </property>
     <property name="title">
      <string>额定值配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>相电压额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="lineVolBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="value">
         <number>220</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>相电流额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="lineCurBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>A</string>
        </property>
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>相功率额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>回路电压额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="loopVolBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>V</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="value">
         <number>220</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>回路电流额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="loopCurBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>A</string>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>回路功率额定值：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="linePowBox">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string>KW</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>7.040000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="loopPowBox">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>KW</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>3.520000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>设备配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备系列：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>语言选择：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>执行板数量</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="breakerBox">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>没有</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>存在</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QComboBox" name="standardBox">
        <item>
         <property name="text">
          <string>标准</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中性</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="lineNumBox">
        <item>
         <property name="text">
          <string>单相</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>三相</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="devSpecBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="loopNumBox">
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>屏幕：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>回路数量：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="vhBox">
        <item>
         <property name="text">
          <string>垂直</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>水平</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>版本类型：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="boardNumBox">
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>断路器</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>相位数：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QComboBox" name="languageBox">
        <item>
         <property name="text">
          <string>中文</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>英文</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="sensorBox">
        <item>
         <property name="text">
          <string>没有</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>存在</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>传感器盒子</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>输出位额定电流值配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="1">
       <widget class="QLabel" name="label_16">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>输出位16A额定值</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="QWidget" name="checkWid" native="true">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="op_1">
           <property name="text">
            <string>1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="op_2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="op_3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="op_4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="op_5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QCheckBox" name="op_6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QCheckBox" name="op_7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QCheckBox" name="op_8">
           <property name="text">
            <string>8</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QCheckBox" name="op_9">
           <property name="text">
            <string>9</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QCheckBox" name="op_10">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item row="0" column="10">
          <widget class="QCheckBox" name="op_11">
           <property name="text">
            <string>11</string>
           </property>
          </widget>
         </item>
         <item row="0" column="11">
          <widget class="QCheckBox" name="op_12">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="op_13">
           <property name="text">
            <string>13</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="op_14">
           <property name="text">
            <string>14</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="op_15">
           <property name="text">
            <string>15</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="op_16">
           <property name="text">
            <string>16</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="op_17">
           <property name="text">
            <string>17</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QCheckBox" name="op_18">
           <property name="text">
            <string>18</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QCheckBox" name="op_19">
           <property name="text">
            <string>19</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QCheckBox" name="op_20">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QCheckBox" name="op_21">
           <property name="text">
            <string>21</string>
           </property>
          </widget>
         </item>
         <item row="1" column="9">
          <widget class="QCheckBox" name="op_22">
           <property name="text">
            <string>22</string>
           </property>
          </widget>
         </item>
         <item row="1" column="10">
          <widget class="QCheckBox" name="op_23">
           <property name="text">
            <string>23</string>
           </property>
          </widget>
         </item>
         <item row="1" column="11">
          <widget class="QCheckBox" name="op_24">
           <property name="text">
            <string>24</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="op_25">
           <property name="text">
            <string>25</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="op_26">
           <property name="text">
            <string>26</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="op_27">
           <property name="text">
            <string>27</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QCheckBox" name="op_28">
           <property name="text">
            <string>28</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="op_29">
           <property name="text">
            <string>29</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QCheckBox" name="op_30">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QCheckBox" name="op_31">
           <property name="text">
            <string>31</string>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <widget class="QCheckBox" name="op_32">
           <property name="text">
            <string>32</string>
           </property>
          </widget>
         </item>
         <item row="2" column="8">
          <widget class="QCheckBox" name="op_33">
           <property name="text">
            <string>33</string>
           </property>
          </widget>
         </item>
         <item row="2" column="9">
          <widget class="QCheckBox" name="op_34">
           <property name="text">
            <string>34</string>
           </property>
          </widget>
         </item>
         <item row="2" column="10">
          <widget class="QCheckBox" name="op_35">
           <property name="text">
            <string>35</string>
           </property>
          </widget>
         </item>
         <item row="2" column="11">
          <widget class="QCheckBox" name="op_36">
           <property name="text">
            <string>36</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="op_37">
           <property name="text">
            <string>37</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="op_38">
           <property name="text">
            <string>38</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="op_39">
           <property name="text">
            <string>39</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QCheckBox" name="op_40">
           <property name="text">
            <string>40</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="op_41">
           <property name="text">
            <string>41</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QCheckBox" name="op_42">
           <property name="text">
            <string>42</string>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QCheckBox" name="op_43">
           <property name="text">
            <string>43</string>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QCheckBox" name="op_44">
           <property name="text">
            <string>44</string>
           </property>
          </widget>
         </item>
         <item row="3" column="8">
          <widget class="QCheckBox" name="op_45">
           <property name="text">
            <string>45</string>
           </property>
          </widget>
         </item>
         <item row="3" column="9">
          <widget class="QCheckBox" name="op_46">
           <property name="text">
            <string>46</string>
           </property>
          </widget>
         </item>
         <item row="3" column="10">
          <widget class="QCheckBox" name="op_47">
           <property name="text">
            <string>47</string>
           </property>
          </widget>
         </item>
         <item row="3" column="11">
          <widget class="QCheckBox" name="op_48">
           <property name="text">
            <string>48</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="uncheckBtn">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>全部取消勾选</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
